src/router.rs

```diff
@@ -100,11 +100,16 @@ impl RouteInfo {
     /// would return `false`.
```

Comment:
If you are going to update the method to handle prefixes, you'd need to update documentation.